docs/reference/troubleshooting/common-issues/high-cpu-usage.asciidoc

```diff
@@ -9,12 +9,30 @@ If a thread pool is depleted, {es} will <<rejected-requests,reject requests>>
 related to the thread pool. For example, if the `search` thread pool is
 depleted, {es} will reject search requests until more threads are available.
 
+CPU degradation frequently occurs related to a <<data-tiers,data tier>>'s traffic,
+potentially being <<hotspotting,hot spotted>>.
```

Comment:
this sentence is confusing to me. couple of things:

- the cpu isn't degraded, is it? it's just experiencing high usage? 
- data tier stuff needs a little clarification. perhaps:

```suggestion
You might experience high CPU usage if a <<data-tiers,data tier>>, and therefore the nodes assigned to that tier, is experiencing more traffic than other tiers. This imbalance in resource utilization is also known as <<hotspotting,hot spotting>>.
```